service/awsconfig/v2/cloudconfig/templates.go

```diff
@@ -210,7 +210,7 @@ Before=set-ownership-etcd-data-dir.service etcd3.service
 
 [Mount]
 What=/dev/xvdh
-Where=/etc/kubernetes/data/etcd
+Where=/var/lib/etcd
```

Comment:
Why is this? I think v2 is frozen and should not change. The same change happened for v5 and we have a changelog for the corresponding version bundle. There is no changelog for the version bundle of the corresponding changes for v2 though. This feels super wrong. I am not sure about the implications here. I also think that it would be good to have separate PRs for changes on different versions.

Reply:
Eagle eye Tim :) +1 I think it affects only `v_3_1_1` which shouldn't be imported in `v2`.

Reply:
Indeed, very good catch, this is not needed at all, the only required template change is in `/service/awsconfig/v5/templates/cloudconfig/mount_etcd_volume.go` probably something went wrong with my rebase after the latest template changes, removed.